docs/deployment.rst

```diff
@@ -1,63 +1,94 @@
 Deployment
 ==========
 
-moz-dev-dash is designed with `12-factor app philosophy`_ to run on `heroku`_, so you
-can easily deploy your changes to your own heroku app with `heroku toolbelt`_.
+moz-dev-dash is designed with `12-factor app philosophy`_, so you can easily
+deploy your changes to your own app.
 
 
-Deploy your own
----------------
+Deploy your own (on Mozilla Deis)
+---------------------------------
 
-#. `Create a heroku remote`_. We suggest naming it moz-dev-dash-`username`::
+#. `Install the deis client`_.
 
-    heroku apps:create moz-dev-dash-username
+#. `Register`_/login with the Mozilla Deis controller::
 
-#. Set the heroku app to use the "multi" buildpack::
+    deis register http://deis.deis.dev.mozaws.net
+    deis auth:login http://deis.deis.dev.mozaws.net
 
-    heroku buildpacks:set https://github.com/ddollar/heroku-buildpack-multi.git
+#. Add your public key::
 
-#. Push code to the heroku remote::
+    deis keys:add
 
-    git push heroku master
+#. Create the application::
 
-#. `Migrate`_ DB tables on heroku::
+    deis create dev-dashboard-username
 
-    heroku run python manage.py migrate
+#. Push code to the deis remote::
 
-#. Create a superuser on heroku::
+    git push deis master
 
-    heroku run python manage.py createsuperuser
+#. `Request a dev IAM account from Mozilla Cloud Ops`_ to create your RDS
+   Postgres instance in the Mozilla account.
 
-#. Open the new heroku app::
+#. `Create an RDS Postgres instance`_ in us-east-1 with default settings except:
```

Comment:
I can link straight to https://console.aws.amazon.com/rds/home?region=us-east-1#launch-dbinstance:ct=dashboard:, but there's still a bit of Amazon RDS you have to learn ... did you find a helpful link/article for getting started with it?